recipes/service.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Cookbook: rundeck-bridge
+# Recipe:   service
+#
+
+# Compute final command with every options (even from Wrapper cookbooks)
+cmd = [ node['rundeck_bridge']['binary'] ]
+node['rundeck_bridge']['options'].each do |key, value|
+ cmd << "--#{key} #{value}"
+end
+command = cmd.join(' ')
+
+
+# Overwrite and update the command
+# (Workaround for Upstart and logs)
+node['rundeck_bridge']['poise_service']['options'].each do |k, v|
```

Comment:
I have tried, but I faced some issue with the wrapper cookbook we use @criteo (`rundeck-central`).
The `cmd` is "overwrittent" or completed by a bunch of options that need to be "computed" at compilation time. That is why I moved it in a recipe.